src/System.CommandLine.Tests/SuggestionTests.cs

```diff
@@ -997,6 +993,40 @@ public void When_there_are_multiple_arguments_then_suggestions_are_only_offered_
             {
                 Assert.True(false, "Test testname is not written yet.");
             }
+
+            [Fact]
+            public void Enum_suggestions_can_be_configured_with_list_clear()
+            {
+                var argument = new Argument<LogLevel?>();
+                argument.SuggestionSources.Clear();
+                argument.SuggestionSources.Add(new[] { "q", "quiet", "m", "minimal", "n", "normal", "d", "detailed", "diag", "diagnostic" });
+                var command = new Command("the-command")
+                {
+                    argument
+                };
+
+                var suggestions = command.Parse("the-command d")
+                                         .GetSuggestions();
+
+                suggestions.Should().BeEquivalentTo("d", "detailed", "diag", "diagnostic");
+            }
+
+            [Fact]
+            public void Enum_suggestions_can_be_configured_without_list_clear()
+            {
+                var command = new Command("the-command")
+                {
+                    new Argument<LogLevel?>()
+                    {
+                        SuggestionSources = { "q", "quiet", "m", "minimal", "n", "normal", "d", "detailed", "diag", "diagnostic" }
```

Comment:
Seeing this written this way, it looks like it would read much more naturally as:

```cs
Suggestions = { "q", "quiet", "m", "minimal", "n", "normal", "d", "detailed", "diag", "diagnostic" }
```